src/pytesseract.py

```diff
@@ -348,6 +354,9 @@ def image_to_data(image,
 
     if output_type == Output.DICT:
         return file_to_dict(run_and_get_output(*args), '\t', -1)
+    elif pandas_installed and output_type == Output.DATAFRAME:
```

Comment:
Can we refactor this a bit. We need something like this:
```python
 class PandasNotSupported(EnvironmentError):
     def __init__(self):
         super(PandasNotSupported, self).__init__('Missing pandas package')

# ....

elif output_type == Output.DATAFRAME:
    if not pandas_installed:
        raise PandasNotSupported()

    args.append(True)
    return pd.read_csv(StringIO(run_and_get_output(*args)), sep="\t")
```

This way the user will know that the requested output is not supported with the current environment.